tests/python_client/testcases/test_search.py

```diff
@@ -2034,6 +2037,65 @@ def test_search_expression_all_data_type(self, nb, nq, dim, auto_id, _async):
         assert (default_int64_field_name and default_float_field_name and default_bool_field_name) \
                in res[0][0].entity._row_data
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_search_with_comparative_expression(self, _async):
+        """
+        target: test search using comparison
+        method: test search using comparison
+        expected: search success
+        """
+        #1. create a collection
+        nb = 10
+        dim = 1
+        int64_1 = ApiFieldSchemaWrapper().init_field_schema(name="int64_1", dtype=DataType.INT64,
+                                                            is_primary=True)[0]
+        int64_2 = ApiFieldSchemaWrapper().init_field_schema(name="int64_2", dtype=DataType.INT64,
+                                                            is_primary=False)[0]
+        fields = [int64_1, int64_2, cf.gen_float_vec_field(dim=dim)]
+        schema = ApiCollectionSchemaWrapper().init_collection_schema(fields=fields,primary_field="int64_1")[0]
+        collection_w = self.init_collection_wrap(name="comparison", schema=schema)
+
+        #2. inset data
+        values = pd.Series(data=[i for i in range(0, nb)])
+        dataframe = pd.DataFrame({"int64_1": values, "int64_2": values,
+                           ct.default_float_vec_field_name: cf.gen_vectors(nb, dim)})
+        collection_w.insert(dataframe)
+
+        #3. search with expression
+        collection_w.load()
+        expression = "int64_1 <= int64_2"
+        vectors = [[random.random() for _ in range(dim)] for _ in range(default_nq)]
+        collection_w.search(vectors[:nq], default_search_field,
+                            default_search_params, default_limit,
+                            expression, _async=_async,
+                            check_task=CheckTasks.check_search_results,
+                            check_items={"nq": nq,
+                                         "limit": default_limit,
+                                         "_async": _async})
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_search_with_expression_invalid_like(self, dim, _async):
```

Comment:
This invalid case should  be moved to invalid class.

Reply:
> This invalid case should be moved to invalid class.

The case has been moved.